src/Components/ObjectProperties/ObjectProperties.js

```diff
@@ -57,6 +59,12 @@ class ObjectProperties extends React.Component {
     this.subscription = this.context.eventEmitter.addListener('alignKeys', this.alignKeys);
   };
 
+  componentWillReceiveProps = (nextProps) => {
+    if (nextProps.keyWidth) {
+      this.setState({ keyWidth: nextProps.keyWidth });
```

Comment:
I am afraid this is an anti pattern - you could potentially trigger the rendering function twice. What's the aim here?